src/colornames.csv

```diff
@@ -27976,6 +28066,7 @@ Veiled Treasure,#f6edb6,x
 Veiled Violet,#b19bb0,
 Veiling Waterfalls,#d4eaff,x
 Velddrif,#a17d61,
+Veldrift,#a68468,

```

Comment:
Did not find anything on this color. Lets remove it. Could be an inside joke or something :D